mtest/libmscore/midi/tst_midi.cpp

```diff
@@ -378,6 +378,7 @@ void TestMidi::events()
       QTextStream out(&filehandler);
 
       for (auto iter = events.begin(); iter!= events.end(); ++iter){
+            if (iter->second.discard()) continue;
```

Comment:
When rebasing, please put `continue` at the next line.

Reply:
Done in rebasing.